### ISSUE TITLE
Error while running "npm run blockit"

### ISSUE THREAD

**Issue by fintechee** [mudza09/blockit-builder#1]
Hi,

I encountered an error while running "npm run blockit".
TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at F.hookSections (file:///home/marksea/equity/node_modules/blockit-builder/blockit.js:15:1500)
    at ee (file:///home/marksea/equity/node_modules/blockit-builder/blockit.js:17:8023)
    at se (file:///home/marksea/equity/node_modules/blockit-builder/blockit.js:17:8091)
    at file:///home/marksea/equity/node_modules/blockit-builder/blockit.js:17:8096
    at ModuleJob.run (node:internal/modules/esm/module_job:198:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:409:24)
    at async loadESM (node:internal/process/esm_loader:85:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)

Node.js v17.9.1

**Comment by mudza09** [comment]
Did you run command "npm install" before "npm run blockit"?

"npm install" is needed to install the dependencies required by blockit, if this is not done then an error will occur

**Comment by fintechee** [comment]
Thank you for your reply.

Yes, I ran "npm install" before running blockit.

**Comment by fintechee** [comment]
I know the reason.
process.env.npm_package_title is only supported by nodejs v18+.
So, replacing it with the package title will solve this issue.

**Comment by Samonnite** [comment]
TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at w.hookSections (file:///Users/zhangqiang/Downloads/Telegram%20Desktop/equity%203/node_modules/blockit-builder/blockit.js:16:1144)
    at se (file:///Users/zhangqiang/Downloads/Telegram%20Desktop/equity%203/node_modules/blockit-builder/blockit.js:20:517)
    at te (file:///Users/zhangqiang/Downloads/Telegram%20Desktop/equity%203/node_modules/blockit-builder/blockit.js:20:586)
    at file:///Users/zhangqiang/Downloads/Telegram%20Desktop/equity%203/node_modules/blockit-builder/blockit.js:20:591
    at ModuleJob.run (node:internal/modules/esm/module_job:192:25)
    at async DefaultModuleLoader.import (node:internal/modules/esm/loader:246:24)
    at async loadESM (node:internal/process/esm_loader:40:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.3.0

why i console.log(process.env.npm_package_title) is undefined

**Comment by mudza09** [comment]
May i know what version of blockit-builder and what theme are you using?

**Comment by Samonnite** [comment]
```
{
  "name": "equity",
  "title": "Equity",
  "version": "1.1.5",
  "description": "Investment and Forex HTML Template",
  "private": true,
  "scripts": {
    "build": "node ./node_modules/blockit-builder/blockit.js --build",
    "blockit": "node ./node_modules/blockit-builder/blockit.js"
  },
  "author": "mudza09",
  "license": "MIT",
  "dependencies": {
    "@fortawesome/fontawesome-free": "6.4.0",
    "particles.js": "2.0.0",
    "share-buttons": "1.9.0",
    "uikit": "3.16.17"
  },
  "devDependencies": {
    "blockit-builder": "^3.2.2"
  }
}

```
I change process.env.npm_package_title to  process.env.npm_package_name  in /node_modules/blockit-builder/blockit.js:16:1144), and resolve it

**Comment by mudza09** [comment]
already fix the issue in v3.2.6
